js/src/components/query/QueryDiffResultView.tsx

```diff
@@ -119,22 +146,27 @@ export const QueryDiffResultView = ({
         gap="5px"
         alignItems="center"
         px="10px"
-        bg={warning ? "orange.100" : "inherit"}
+        bg={warning || invalidPKeyMessage ? "orange.100" : "inherit"}
```

Comment:
Please use the same variable naming

warning -> waringLimit
invalidPKeyMessage -> warningPK